[Cloudflare Logpush] update documentation for s3 and sqs (#5687)
elastic/integrations: packages/cloudflare_logpush/docs/README.md

For example, you could use the data from this integration to know which websites have the highest traffic, which areas have the highest network traffic, or observe mitigation statistics.

> ## Logpush Data Sets

Reviewer: Please revert this. The current state is conventional wording.

## Setup

### To collect data from AWS S3 Bucket, follow the below steps:
- Configure [Cloudflare Logpush to Amazon S3](https://developers.cloudflare.com/logs/get-started/enable-destinations/aws-s3/) to send data to an AWS S3 bucket.
- The default values of the "Bucket Prefix" for each Logpush data set are listed below. However, the user can set the parameter "Bucket Prefix" for each Logpush data set according to their requirements.

>   | Logpush Data Set Name | Bucket Prefix          |

Reviewer: Why are these changes needed?

  | --------------------- | ---------------------- |
  | Audit Logs            | audit_logs             |
  | DNS                   | dns                    |
  | Firewall Event        | firewall_event         |
  | HTTP Request          | http_request           |
  | NEL Report            | nel_report             |
  | Network Analytics     | network_analytics_logs |
  | Spectrum Event        | spectrum_event         |

### To collect data from AWS SQS, follow the below steps:
1. If Logpush forwarding to an AWS S3 Bucket hasn't been configured, then first setup an AWS S3 Bucket as mentioned in the above documentation, and enable forwarding for the relevant account-level or zone-level Logpush data sets.
2. Follow the steps below for each Logpush data set that has been enabled:
    1. Create an SQS queue
        - To setup an SQS queue, follow "Step 1: Create an Amazon SQS queue" mentioned in the [Documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ways-to-add-notification-config-to-bucket.html).
        - While creating an SQS Queue, please provide the same bucket ARN that has been generated after creating an AWS S3 Bucket.

>     2. Setup event notification from the S3 bucket. Follow this [Link](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications.html). Use the following settings:

Reviewer: ```suggestion
    2. Setup event notification from the S3 bucket according to the documentation [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications.html). Use the following settings:
```
